src/hooks/useCds/convert-strides.mjs

```diff
@@ -0,0 +1,25 @@
+import { writeFileSync } from 'fs';
+import { join } from 'path';
+import csvToJson from 'convert-csv-to-json';
+
+let keyColumn = 'MRN';
+let groupedResults = {};
+let csvFile = 'data/strides.csv';
+
+if (process.argv.length === 3) {
+  csvFile = process.argv[2];
+}
+
+const jsonFile = join('src/hooks/useCds/strides.js');
+const results = csvToJson.fieldDelimiter(',').supportQuotedField(true).getJsonFromCsv(csvFile);
```

Comment:
Should we add more graceful handling for when a CSV file is not available in the supplied directory?